app/controllers/application_controller.rb

```diff
@@ -1,5 +1,29 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
+  # def current_user
+  #   User.first
+  # end
+
```

Comment:
- [ ] [OPTIONAL] Kindly remove the current user completely instead of commenting it out.